gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Wed Jun 20 13:31:09 EDT 2018
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.8-bin.zip
+distributionBase=PROJECT
+distributionPath=build/offline/gradle/wrapper/dists
+zipStoreBase=PROJECT
+zipStorePath=build/offline/gradle/wrapper
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8-bin.zip
```

Comment:
gradlew kept insisting on downloading it from internet. seems like it looks in gradle-home for dists, despite the PROJECT setting.. I resorted to changing url to just the zip file name, and dropping a copy of the zip next to gradle/wrapper/gradle-wrapper.jar